website/docs/r/cognitive_account_ai_services.html.markdown

```diff
@@ -0,0 +1,150 @@
+---
+subcategory: "Cognitive Services"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cognitive_account_ai_services"
+description: |-
+  Manages a Cognitive AI Services Account.
+---
+
+# azurerm_cognitive_account_ai_services
+
+Manages a Cognitive AI Services Account.
```

Comment:
Please note "Cognitive Services" was rebranded to Azure AI services" a year ago. 
```suggestion
Manages an Azure AI Services Account.
```

Reply:
There are other occurrences to replace terminology in this file and maybe other places.

Reply:
Thanks for reviewing and the naming suggestion @eric-urban.

Regarding the name of the resource, I would suggest going with `azurerm_ai_services` since it's implicit that we're dealing with azure here, no other resource in the provider specifies azure in addition to the resource type. Is this acceptable for you?